migrations/20240717123726_add_sms_mfa_config.up.sql

```diff
@@ -0,0 +1,11 @@
+do $$ begin
```

Comment:
This needs to be adjusted so that we can create factor type even though the existing column has factor_type set to ('totp', 'webauthn') only

Reply:
Please use `ALTER TYPE ADD NEW VALUE ...` instead: https://www.postgresql.org/docs/current/sql-altertype.html